test/classes/initializers/nested/scoping-mixed.chpl

```diff
@@ -0,0 +1,28 @@
+
+// This test exists to ensure that we do not encounter weird scoping issues
+// when a top-level type has a constructor and a nested type has an
+// initializer. This case did not 'just work' when initializers were enabled
+// for nested types and required compiler changes.
+
+class Node {
+  var x : int;
+}
```

Comment:
Does this test need a --no-force-initializers (or maybe always run both with --force-initializers and --no-force-initializers) ?

Reply:
This test passes successfully with and without --force-initializers.